src/samplosf/src/windict/compile_dict.c

```diff
@@ -263,6 +263,15 @@ void pushEntry();
 unsigned char nonWhite();
 unsigned char getAlpha();
 int look_for_prefix(char, char);
+int get_entry(char*);
+int parseString(char*);
+void scan_entry();
+void sort_entry();
+void scan_error(char*);

```

Comment:
This is correct, but causes an error in line 844 later on as scan_error is called there with a vararg argument. Can you fix line 844 to only call with one arg (maybe sprintf to a temporary buffer or so)?

Reply:
It seemed to build with that, so I just left it :D I've added a sprintf there.